src/mgr/StoreToCommWriter.cc

```diff
@@ -142,7 +144,7 @@ Mgr::StoreToCommWriter::swanSong()
             storeUnregister(sc, entry, this);
             sc = NULL;
         }
-        entry->unregisterAbort();
+        entry->unregisterAbort("StoreToCommWriter object destructed");
```

Comment:
We probably should not imply that the job has been destructed already. It has not, and calling the swanSong() method itself is not really a guarantee of the immediate job destruction. It is (well, should be) more of a guarantee of no more asynchronous job notifications and such.

```suggestion
        entry->unregisterAbort("StoreToCommWriter gone");
```